getting-started/kubernetes/vpp/baremetal.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Install VPP on a baremetal cluster
```

Comment:
Please add the "tech preview" note to the first topic in every section in the TOC for VPP. We don't know the entry point for users across so many VPP docs. 

> **Warning!** The VPP dataplane is a tech preview and should not be used in production clusters. It has had limited testing and it will contain bugs (please report these on the [Calico Users slack](https://calicousers.slack.com/archives/C017220EXU1) or [Github](https://github.com/projectcalico/vpp-dataplane/issues)).  In addition, it does not support all the features of {{site.prodname}} and it is missing some security features such as Host Endpoint policies.
{: .alert .alert-danger }